crates/db_views/src/review_comment_view.rs

```diff
@@ -0,0 +1,108 @@
+use crate::structs::ReviewCommentView;
+use diesel::{result::Error, ExpressionMethods, JoinOnDsl, NullableExpressionMethods, QueryDsl};
+use diesel_async::RunQueryDsl;
+use lemmy_db_schema::{
+  schema::{comment, community, person, post, review_comment},
+  source::{
+    comment::Comment,
+    community::Community,
+    person::Person,
+    post::Post,
+    review_comment::ReviewComment,
+  },
+  traits::JoinView,
+  utils::{get_conn, limit_and_offset, DbPool},
+};
+use typed_builder::TypedBuilder;
+
+type ReviewCommentViewTuple = (
+  ReviewComment,
+  Comment,
+  Post,
+  Community,
+  Person,
+  Option<Person>,
+);
+
+impl ReviewCommentView {
+  /// Returns the current unresolved review count
+  pub async fn get_review_count(pool: &DbPool) -> Result<i64, Error> {
+    use diesel::dsl::count;
+
+    let conn = &mut get_conn(pool).await?;
+
+    review_comment::table
```

Comment:
This doesn't need any data from joins. It should be on the `CommentReview` impl.